tests/JustSaying.UnitTests/Messaging/Channels/SubscriptionGroupCollectionTests.cs

```diff
@@ -108,10 +115,9 @@ private static FakeSqsQueue TestQueue(
             QueueName = queueName,
         };
 
-        var messages = new List<Message>
-        {
-            new TestMessage { Body = messageSerializationRegister.Serialize(message, false) },
-        };
+        var messageConverter = new PublishMessageConverter(PublishDestinationType.Queue, new NewtonsoftMessageBodySerializer<TestJustSayingMessage>(), new MessageCompressionRegistry(), new PublishCompressionOptions(), "TestJustSayingMessage", false);
```

Comment:
Feels like there's a test helper method that can be added somewhere to save all these really long lines of boilerplate.

Reply:
I've tried to move these into base classes, it's all still a bit ugly as some of the setup varies per test